maps/hazard_pits.go

```diff
@@ -79,7 +80,7 @@ func (m HazardPitsMap) SetupBoard(initialBoardState *rules.BoardState, settings
 		}
 	}
 
-	err := rules.PlaceFoodFixed(rand, tempBoardState)
+	err := PlaceFoodFixed(rand, tempBoardState, tempEditor)
```

Comment:
@xtagon any idea why it's suggesting this change?

Reply:
I made that change to make the lint pass, it said the old way was deprecated